### DISCUSSION TITLE
State of React Suspense

### DISCUSSION THREAD

**Discussion by adrianschneider94** [urql-graphql/urql#1629] [Q&A] [answered]
Hi there,

what's the status of React Suspense mode at the moment?
[The old npm package page](https://www.npmjs.com/package/@urql/exchange-suspense) states that client side suspense isn't recommended anymore. However seeing pull requests and issues it looks like it's still in development.

In the code I could see, that the preact bindings include suspense code while the react bindings don't.

Am I right to assume, that the development is going on within preact until it's stable and then transferred tot the react bindings? Or what's the state/plan?

Thanks for your work btw!

**Comment by JoviDeCroock** [comment, marked as answer]
Both React and Preact have integrated Suspense support in the core-package, we moved away from experimental support by deprecating the old npm package.
